code/__HELPERS/maths.dm

```diff
@@ -241,3 +241,8 @@ proc/arctan(x)
 	var/mult = input/scale
 	var/trinum = (sqrt(8 * mult + 1) - 1 ) / 2
 	return trinum * scale
+
+/proc/round_up(num)
```

Comment:
Is it? I saw Ceiling but couldn't quickly make sense of it, and found this on the byond forum.